MomentumDiscordBot/Commands/General/GeneralModule.cs

```diff
@@ -10,11 +12,17 @@ public class GeneralModule : MomentumModuleBase
     {
         public Configuration Config { get; set; }
 
-        [Command("key")]
-        [Description("Give information on obtaining a game key")]
-        public async Task KeyBeggingResponseAsync(CommandContext context)
+        [SlashCommand("key", "Give information on obtaining a game key")]
+        public async Task KeyBeggingResponseAsync(InteractionContext context)
         {
-            await ReplyNewEmbedAsync(context, Config.KeyBeggingResponse, MomentumColor.Blue);
+            await SlashReplyNewEmbedAsync(context, Config.KeyBeggingResponse, MomentumColor.Blue);
         }
+
+        //[Command("key")]
+        //[Description("Give information on obtaining a game key")]
+        //public async Task KeyBeggingResponseAsync(CommandContext context)
+        //{
+        //    await ReplyNewEmbedAsync(context, Config.KeyBeggingResponse, MomentumColor.Blue);
+        //}
```

Comment:
Don't included commented code blocks

Reply:
This was done as I'm not able to test without a configuration file; and I do not have api keys for everything listed.